identity/client/src/pages/tenants/detail/index.tsx

```diff
@@ -119,6 +121,15 @@ const Details: FC = () => {
                       },
                     ]
                   : []),
+                ...(IS_TENANT_CLIENTS_SUPPORTED && isOIDC
```

Comment:
I think for now this is ok so the structure doesn't get unnecessarily convoluted, but when the feature flag gets removed this tab should be together with the mappings one in the array above to avoid duplication of the `isOIDC` conditional.